src/main/java/com/springbootWithJPA/springBootRestExample/EmployeeController.java

```diff
@@ -1,59 +1,49 @@
-package com.springbootWithJPA.springBootRestExample;
+package com.springbootwithjpa.springbootrestexample;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.DeleteMapping;
-
 import java.util.List;
-import java.util.Optional;
 
 @RestController
 @RequestMapping("/api")
 public class EmployeeController {
-
     @Autowired
     private EmployeeRepository employeeRepository;
 
-    @PostMapping("/employee")
+    @Autowired
+    private EmployeeService employeeService;
+
+    @PostMapping("/createEmployee")
```

Comment:
When you create endpoints related to same resource, you should have the noun as the mapping with GET and POST differentiating what they do. For eg if we create employees, the requests should be

/employee (POST) to create
/employee/{id} (PUT) to update
/employee/{id} (GET) to get an employee
/employees (GET) list all employees
/employee/{id} (DELETE) delete an employee

Reply:
addressed

Reply:
Where did you address this

Reply:
https://github.com/Jashika/springBootRestExample/pull/1/commits/f47f8da73d8ce44e23c658b0ab71adab38741869.I dint create another feature branch i did a mistake

Reply:
https://github.com/Jashika/springBootRestExample/pull/1/commits/f47f8da73d8ce44e23c658b0ab71adab38741869